src/scicloj/clay/v2/server.clj

```diff
@@ -2,17 +2,18 @@
   (:require
    [clojure.java.browse :as browse]
    [clojure.java.io :as io]
-   [clojure.java.shell :as shell]
    [clojure.string :as string]
    [hiccup.page]
    [org.httpkit.server :as httpkit]
    [scicloj.clay.v2.server.state :as server.state]
-   [scicloj.clay.v2.util.path :as path]
    [scicloj.clay.v2.util.time :as time]
    [scicloj.clay.v2.item :as item]
-   [scicloj.kindly.v4.api :as kindly]
    [clojure.string :as str]
-   [hiccup.core :as hiccup]))
+   [hiccup.core :as hiccup])
+  (:import (java.io FileInputStream)
+           (java.net ServerSocket)))
+
+(set! *warn-on-reflection* true)
```

Comment:
I've deleted it to avoid confusion,

just for reference: https://github.com/clojure/clojure/blob/8c8e4f6ef21f3d0f59deb60cdc7e1b584f596d59/src/jvm/clojure/lang/Compile.java#L62

There are 3 special bindings that get reset to their previous state after loading a Clojure file
compile_path, warn_on_reflection, unchecked_math, uncheckedMath

For this reason when you load a namespace with `set! *warn-on-reflection*` in it, it does not change.